wb_vandalism/datasources/revision_metadata.py

```diff
@@ -0,0 +1,10 @@
+
+from revscoring.datasources import Datasource
+from revscoring.datasources.revision import metadata
+
+
+def process_comment(metadata):
```

Comment:
Mostly I'm thinking of dependency injection while we extract features from dumps. Initially I passed metadata but then I changed my mind since sending metadata makes extracting features from dumps almost impossible.